robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAccessibilityNodeInfo.java.vm

```diff
@@ -472,6 +667,140 @@ public class ShadowAccessibilityNodeInfo {
   }
 
   @Implementation
+  public int getMovementGranularities() {
+    return movementGranularities;
+  }
+
+  @Implementation
+  public void setMovementGranularities(int movementGranularities) {
+    this.movementGranularities = movementGranularities;
+  }
+
+  @Implementation
+  public CharSequence getPackageName() {
+    return packageName;
+  }
+
+  @Implementation
+  public void setPackageName(CharSequence packageName) {
+    this.packageName = packageName;
+  }
+
+#if ($api >= 18)
+  @Implementation
+  public String getViewIdResourceName() {
+    return viewIdResourceName;
```

Comment:
Technically this is dependent on the capabilities set by the AccessibilityService caller's AccessibilityServiceInfo's accessibilityFlags value.  If it doesn't include AccessibilityServiceInfo.FLAG_REPORT_VIEW_IDS, this will always be null.

Having thought about this though, the AccessibilityManager implementation in RL probably doesn't track flags used, so I very much doubt there's anything we can do here. :-(